spec/index.bs

```diff
@@ -3640,7 +3703,6 @@ A {{GPUBindGroup}} object has the following internal slots:
             **Returns:** {{GPUBindGroup}}
 
             1. Let |bindGroup| be a new valid {{GPUBindGroup}} object.
-            1. Let |limits| be |this|.{{GPUObjectBase/[[device]]}}.{{device/[[limits]]}}.{{supported limits/maxUniformBufferBindingSize}}.
```

Comment:
why is this removed?

Reply:
Looks like this line was supposed to be edited instead of removed? (to remove the maxUniformBufferBindingSize from the end)